pom.xml

```diff
@@ -892,7 +907,25 @@
       <dependency>
         <groupId>org.scalatest</groupId>
         <artifactId>scalatest_${scala.binary.version}</artifactId>
-        <version>3.0.8</version>
+        <version>3.2.0</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.scalatestplus</groupId>
+        <artifactId>scalatestplus-scalacheck_${scala.binary.version}</artifactId>
+        <version>3.1.0.0-RC2</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.scalatestplus</groupId>
+        <artifactId>scalatestplus-mockito_${scala.binary.version}</artifactId>
+        <version>1.0.0-SNAP5</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.scalatestplus</groupId>
+        <artifactId>scalatestplus-selenium_${scala.binary.version}</artifactId>
+        <version>1.0.0-SNAP5</version>
```

Comment:
Oh, interesting. All of these are `test dependency` changes, but `dev/test-dependencies.sh` seems to complain for some reasons.
```
--- a/dev/deps/spark-deps-hadoop-2.7-hive-1.2
+++ b/dev/pr-deps/spark-deps-hadoop-2.7-hive-1.2
@@ -157,9 +157,9 @@ metrics-json/4.1.1//metrics-json-4.1.1.jar
 metrics-jvm/4.1.1//metrics-jvm-4.1.1.jar
 minlog/1.3.0//minlog-1.3.0.jar
 netty-all/4.1.47.Final//netty-all-4.1.47.Final.jar
-objenesis/2.5.1//objenesis-2.5.1.jar
+objenesis/2.6//objenesis-2.6.jar
 okhttp/3.12.6//okhttp-3.12.6.jar
-okio/1.15.0//okio-1.15.0.jar
+okio/1.14.0//okio-1.14.0.jar
```

Reply:
Weird, yeah. Updating objenesis is probably fine. Downgrading okio (?) maybe not.

- hadoop-hdfs-client pulls in okhttp 2.7.5 which depends on okio 1.6.0
- io.fabric8 (used in K8S) pulls in okhttp 3.12.6 and okio 1.15.0
- scalatest-selenium (new) pulls in okhttp 3.11.0 and okio 1.14.0

We actually have both okhttp 2 and 3 in the Hadoop 3 deps! maybe this is OK
It may be that it's OK for the assembly to land on 1.14.0 - managed up much more from Hadoop's version than down from the fabric8 version. That fabric8 update happened relatively recently, but in 3.0.0:
https://github.com/apache/spark/commit/78bdcfade1898ecbde3db4542cac6b29f88eb26c

We can manage it (and okhttp) explicitly but is it worth it?

Well I'll push the dep updates anyway. Looks like it passed?

Reply:
Oh. Got it.